exasol_integration_test_docker_environment/lib/test_environment/abstract_spawn_test_environment.py

```diff
@@ -127,7 +127,7 @@ def _start_database(self, attempt) \
         is_database_ready = yield from self._wait_for_database(database_info, attempt) # type: ignore
         return network_info, database_info, is_database_ready, test_container_info # type: ignore
 
-    def _create_ssl_certificates(self) -> Generator:
+    def _create_ssl_certificates(self) -> Generator[BaseTask, None, Optional[DockerVolumeInfo]]:
```

Comment:
Return Type and send Type of the Generator Type don't makes sense, the send Type might be an artifact of how Luigi or our framework around it works, but the return type should be probably Optional[SshInfo]

Reply:
Or alternatively the name ssl_info in this function is not chosen well

Reply:
Take a look at the method avobe. I think the return type is correct. However, I will rename it.